python/packages/autogen-ext/src/autogen_ext/agents/__init__.py

```diff
@@ -1,4 +1,4 @@
 from ._openai_assistant_agent import OpenAIAssistantAgent
-from .web_surfer._multimodal_web_surfer import MultimodalWebSurfer
 
-__all__ = ["MultimodalWebSurfer", "OpenAIAssistantAgent"]
```

Comment:
Why not keep all agents at the top-level?

Reply:
The issue was that different agents have different dependencies, and this was causing for all to be imported?

Reply:
Yes. You are right!